lib/fog/libvirt/compute.rb

```diff
@@ -57,12 +57,16 @@ def initialize(options={})
           require 'libvirt'
         end
 
-        private
-
         def client
-          return @client if defined?(@client)
+          @client ||= Class.new do
+            def libversion
+              1002009
```

Comment:
A test was added to `tests/libvirt/models/compute/server_tests.rb` with the switch to dhcp_leases:
https://github.com/fog/fog-libvirt/blob/558c05d9d6ca6e764e03aa450b57b2728a7f3404/tests/libvirt/models/compute/server_tests.rb#L23-L25

I couldn't find a better way to help that only go through the dhcp_leases code by default. Happy to switch if there is another way, just not familiar enough with shindo testing framework to know how to solve this better.